utils.py

```diff
@@ -38,74 +36,31 @@ def flatten_custom_fields(elements: Sequence[Dict]) -> Dict:
 
 def load_orcid_information(orcids: Iterable[str]) -> Dict[str, Dict]:
     result = {}
-    orcid_api = OrcidAPI()
+    orcid_format_string = "https://orcid.org/{}/person.json"
     for orcid in orcids:
-        result[orcid] = orcid_api.personal_details(orcid_id=orcid)
+        response = requests.get(orcid_format_string.format(orcid))
```

Comment:
Consider to use an f-string literal instead of re-using a format string template. It's actually less total code in this case.
```suggestion
        response = requests.get(f"https://orcid.org/{orcid}/person.json")
```